Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -122,17 +122,137 @@ that we do not rely on a (non-existing) fallback.
 Cilium's BPF kube-proxy replacement is supported in direct routing as well as in
 tunneling mode.
 
-Direct Server Return (DSR) and other NodePort Settings
-######################################################
+Validate the Setup
+##################
 
-When running Cilium's BPF kube-proxy replacement, by default, a NodePort or
-ExternalIPs service will be accessible through the IP address of a native device
-which has the default route on the host. To change the device, set its name in
-the ``global.nodePort.device`` helm option.
+After deploying Cilium with above Quick-Start guide, we can first validate that
+the Cilium agent is running in the desired mode:
 
-In addition, thanks to the :ref:`host-services` feature, the NodePort service can
-be accessed by default from a host or a Pod within a cluster via it's public,
-cilium_host device or loopback address, e.g. ``127.0.0.1:NODE_PORT``.
+.. parsed-literal::
+
+    kubectl exec -it -n kube-system cilium-fmh8d -- cilium status | grep KubeProxyReplacement
+    KubeProxyReplacement:   Strict   [NodePort, ExternalIPs, HostReachableServicesTCP, HostReachableServicesUDP]
```

Comment:
Probably you can already apply the new output from: https://github.com/cilium/cilium/pull/10083.